docs/csharp/tutorials/string-interpolation.md

```diff
@@ -1,122 +1,98 @@
 ---
-title: String Interpolation - C#
-description: Learn how string interpolation works in C# 6
-author: mgroves
-ms.date: 03/06/2017
-ms.assetid: f8806f6b-3ac7-4ee6-9b3e-c524d5301ae9
+title: String interpolation in C#
+description: Learn how to perform common string formatting tasks in C# using the string interpolation feature
+author: pkulikov
+ms.date: 05/07/2018
 ---
+# String interpolation in C# #
```

Comment:
Since you've made the C# examples interactive, the csharp-interactive-note.md file should be included.